packages/admin/cms-admin/src/dam/Table/FolderTable.tsx

```diff
@@ -150,6 +151,7 @@ const FolderTable = ({
     const loading = tableLoading && tableData === undefined;
     const foldersTableData = tableData?.data.filter(isFolder);
     const filesTableData = tableData?.data.filter(isFile);
+    const firstLastUploadedFileId = filesTableData?.find((file) => lastUploadedFileIds.includes(file.id))?.id;
```

Comment:
Why don't you just use `lastUploadedFileIds[0]`?

Reply:
Because the order inside `lastUploadedFileIds` might not be the actual order of the files in the Table